### PR TITLE
ACS-1998 Describe new T-Engine config elements may be used to modify the config

### PR DESCRIPTION
**Pull request by alandavis**
* Try to make changes similar to those on the source page in acs-packaging.

Also see: https://github.com/Alfresco/acs-packaging/pull/2115

Think I will also need to visit the page again having looked at: https://docs.alfresco.com/transform-service/latest/config/

### CODE REVIEW

File: content-services/latest/develop/repo-ext-points/content-transformers-renditions.md [hunk 1]
```diff
@@ -142,35 +174,15 @@ Rendition provides values for options: Op2, Op3
...
-is enabled.
-
-```text
-transform.service.enabled=true
-local.transform.service.enabled=true
-```
-
-Setting the enabled state to **false** will disable all of the transforms performed by that particular service. It is 
-possible to disable individual Local Transforms by setting the corresponding T-Engine URL property `localTransform.<engineName>.url` 
-value to an empty string.
```
Comment (alandavis) [inline content-services/latest/develop/repo-ext-points/content-transformers-renditions.md:164]: Moved further down the page

File: content-services/latest/develop/repo-ext-points/content-transformers-renditions.md [hunk 2]
```diff
@@ -123,6 +125,36 @@ local.transform.service.cronExpression=4 30 0/1 * * ?
...
+```
+
+Setting the enabled state to **false** will disable all of the transforms
+performed by that particular service. It is possible to disable individual
+Local Transforms by setting the corresponding T-Engine URL property
+**localTransform.&lt;engineName>.url** value to an empty string.
+
+```text
+localTransform.helloworld.url=
+```
```
Comment (alandavis) [inline content-services/latest/develop/repo-ext-points/content-transformers-renditions.md:154]: Moved in the page as it make more sense earlier now.

**Review by gravitonian — APPROVED** [review]
Did some smaller edits, but looks good. Haven't verified the technical stuff...

**Comment by gravitonian** [comment]
This content is now part of PR #507

**Pull request closed without merging**